database/factories/TorrentFactory.php

```diff
@@ -26,34 +26,34 @@
         'user_id'  => function () {
             return factory(App\Models\User::class)->create()->id;
         },
-        'imdb'         => $faker->word,
-        'tvdb'         => $faker->word,
-        'tmdb'         => $faker->word,
-        'mal'          => $faker->word,
-        'igdb'         => $faker->word,
+        'imdb'         => $faker->randomNumber(),
+        'tvdb'         => $faker->randomNumber(),
+        'tmdb'         => $faker->randomNumber(),
+        'mal'          => $faker->randomNumber(),
+        'igdb'         => $faker->randomNumber(),
         'type'         => $faker->word,
-        'resolution'   => $faker->randomNumber(),
+        //'resolution'   => $faker->randomNumber(),
```

Comment:
Im confused where it got these....the ones you commented out that is. I don't ever remember pushing those anywhere. Strange.

Reply:
I agree; I have no idea from where Shift initially obtained these keys. The words `resolution` and `bdinfo` appear nowhere in the database schema nor any migrations. Very odd.

Should we just delete them?

Reply:
Looking at the commit history, you first committed those; I assumed you got them from a Shift-generated file set. In any case, let me know if you want me to trash everything that is commented.

Reply:
yeah remove them